src/app/pages/quickMaps/pages/baselineDetails/map-view/map-view.component.scss

```diff
@@ -1,14 +1 @@
-@import 'src/assets/scss/_colour.scss';
-
-mat-tab-group {
-  min-height: 0;
-  flex: 1 1;
-  & ::ng-deep {
-    & .mat-tab-body-wrapper {
-      flex: 1 1;
-      & .mat-tab-body-content > * {
-        height: 100%;
-      }
-    }
-  }
-}
+// @import 'src/assets/scss/_colour.scss';
```

Comment:
are component scss files required any more if they are not used?

Reply:
No content but better left in